tests/lib/RichText/Converter/Render/EmbedTest.php

```diff
@@ -741,6 +741,56 @@ public function providerForTestConvert(): array
                     ['601', '602'],
                 ],
             ],
+            [
+                '<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Note: this can be replaced with indented HEREDOC for readability, but since this is a merge up, it's ok as it is.